testgrid/config.yaml

```diff
@@ -4091,6 +4091,10 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gce-kubedns-performance
   - name: gce-kubedns-performance-nodecache
     test_group_name: ci-kubernetes-e2e-gce-kubedns-performance-nodecache
+  - name: gce-lb-finalizer
+    test_group_name: ci-kubernetes-e2e-gce-lb-finalizer
+    base_options: include-filter-by-regex=\[sig-network\]
+    description: load balancer tests with finalizer protection for master branch
```

Comment:
I think you need to make a testgroup above

Reply:
wait... presubmit passed?

Reply:
Yeah I was about to add a testgroup and found that we no longer need to explicitly add one for periodic job :)
https://github.com/kubernetes/test-infra/blob/5cb1180f818c66c15645a232cbb7381a0d2463f0/testgrid/config.yaml#L53-L60

Reply:
ahhh right... it's year of 8012 now